src/sentry/tasks/weekly_escalating_forecast.py

```diff
@@ -0,0 +1,91 @@
+import logging
+from datetime import datetime
+from typing import Any, Dict, List, Tuple, Union
+
+from django.db import transaction
+
+from sentry.issues.escalating import query_groups_past_counts
+from sentry.issues.escalating_issues_alg import generate_issue_forecast
+from sentry.models import Group, GroupStatus
+from sentry.models.groupforecast import GroupForecast
+from sentry.tasks.base import instrumented_task
+
+BATCH_SIZE = 1000
+GroupForecastTuple = Tuple[Group, List[int]]
```

Comment:
nit: imo, not sure there's much value in defining this vs just using the `Tuple[Group, List[int]]`